### ISSUE TITLE
Incorporate pressure stall improvements

### ISSUE THREAD

**Issue by hammerandtongs** [facebookincubator/oomd#46]
Just to note there are some pressure stall patches past 4.20 that the android team are working on to improve the monitoring latency for handsets/desktops -

https://lwn.net/ml/linux-kernel/20181214171508.7791-1-surenb@google.com/

Lwn wrote it up here https://lwn.net/SubscriberLink/775971/f10dccac9ed9a965/

**Comment by danobi** [comment]
Yeah I've been talking with Johannes about this. It'll be nice for oomd to have edge triggered events (vs polling the system). I've already volunteered to test out patches once they're ready.
